Makefile

```diff
@@ -225,6 +213,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 .PHONY: deploy
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd manifests/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	$(KUSTOMIZE) build manifests/crds | $(KUBECTL) apply -f -
```

Comment:
This is redundant as they are part of manifests/default..

Reply:
It is added to resolve a timing issue between CRD creation and PaasConfig resource instance creation. Locally it was going fine, but on actions there appeared to be a resource issue. PaasConfig CRD wasn't available yet when it was already trying to create the resource. This resolved it, though nasty.